.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -0,0 +1,45 @@
+name: Bug report
+description: Report a bug in Godot
+body:
+
+- type: markdown
+  attributes:
+    value: |
+      - Write a descriptive issue title above.
+      - Search [open](https://github.com/godotengine/godot/issues) and [closed](https://github.com/godotengine/godot/issues?q=is%3Aissue+is%3Aclosed) issues to ensure it has not already been reported.
+      - Verify that you are using a [supported Godot version](https://docs.godotengine.org/en/stable/about/release_policy.html).
+
+- type: input
+  attributes:
+    label: Godot version
+    description: Specify the Git commit hash if using a development or non-official build.
+    placeholder: Example - "3.x.x"
```

I'd give an explicit example otherwise you'll get people actually writing "3.x.x" since that can match the version they use.
```suggestion
    placeholder: Example - "3.3.stable" or "4.0.dev (3041becc6)"
```